scripts/release/mule/sign/linux/amd64/sign.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo
+date "+build_release begin SIGN stage %Y%m%d_%H%M%S"
+echo
+
+WORKDIR="$1"
+
+if [ -z "$WORKDIR" ]
+then
+    echo "WORKDIR variable must be defined."
```

Comment:
It's not the WORKDIR variable that need to be defined.. it's that the first parameter is missing, right ?

Reply:
Hmm, I should change the order of those params.

Reply:
I'll correct the other targets in another PR.

Reply:
Changed.